lib/DDG/Spice/NewYorkTimes.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::NewYorkTimes;
+# ABSTRACT: Details on New York Times best sellers
+
+use strict;
+use DDG::Spice;
+
+spice to => 'https://api.nytimes.com/svc/books/v3/lists/best-sellers/history.json?api-key={{ENV{DDG_SPICE_NewYorkTimes_APIKEY}}}&title=$1';
+
+triggers startend => "new york times", "new york times best seller";
```

Comment:
Also "nyt review" and "nytimes review":
![drake_at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/15654722/a33abc20-2664-11e6-800d-776e3ee24a24.png)